dll/win32/syssetup/lang/es-ES.rc

```diff
@@ -258,7 +258,7 @@ END
 STRINGTABLE
 BEGIN
     IDS_PRODUCTTITLE "Opciones de producto"
-    IDS_PRODUCTSUBTITLE "Puedes seleccionar una opción de producto afectará al comportamiento del sistema."
+    IDS_PRODUCTSUBTITLE "Puedes seleccionar una opción de producto. Afectará al comportamiento del sistema."
     IDS_PRODUCTSERVERNAME "ReactOS como Servidor"
     IDS_PRODUCTWORKSTATIONNAME "ReactOS como Sobremesa"
     IDS_PRODUCTSERVERINFO "El sistema será reconocido como un servidor. Las carpetas privadas ""Mis imágenes"", ""Mis vídeos"" y ""Mi música"" serán independientes de ""Mis documentos""."
```

Comment:
Pequeño comentario de cara a mejoras futuras: Todos estos con la estructura en pasiva tipo `will be recognized` creo que es mejor traducirlos como `se reconocerán`, en vez de `serán reconocidos`. Esto de los calcos lingüísticos siempre ha sido un problema.

Reply:
El tema es que es una pasiva en realidad. No es que "se reconocerán", porque no es reflexivo, sino que "serán reconocidos" por el software y los programas que se ejecuten en él. 
Por eso es pasiva, nada más. El tema es que tampoco quería incluir un sujeto que no está en el original.
A parte esta sección de código no se ha tocado, pero me parece una sugerencia a tener en cuenta, para no subordinar o pasivizar todas las frases.

Reply:
Antes que "reconocerá", pondría "se expondrá" o "se mostrará". Me encajaría más. Porque en realidad, se reconocerá es uno a uno mismo. Es muy ambiguo.

Reply:
Tienes razón, o `aparecerá como`, me vale. Tampoco quiero ser un pesado por pequeñeces.